x-pack/plugins/enterprise_search/server/lib/pipelines/get_pipeline.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { IngestGetPipelineResponse } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
```

Comment:
@TattdCodeMonkey I changed these imports because typescript was complaining the types weren't matching and this seemed to fix it... I don't _think_ this should be an issue, but want to call it out specifically.

Reply:
AFAIK, both export the same types, and body ones supposed to be deprecated.